### DISCUSSION TITLE
这个情况如何解决

### DISCUSSION THREAD

**Discussion by sexyfeifan** [monlor/docker-xiaoya#203] [General]
![CleanShot 2024-07-15 at 21 55 05@2x](https://github.com/user-attachments/assets/979246cc-d989-4acb-a2e5-86544cdf90c2)
这是因为刚装完?需要手动扫描媒体库么？

**Comment by monlor** [comment]
需要手动扫描

**Reply by sexyfeifan** [reply]
好的 所以这个扫描过程会很漫长是么？
